src/Infrastructure.EntityFramework/AdminConsole/Repositories/PolicyRepository.cs

```diff
@@ -56,6 +56,8 @@ public PolicyRepository(IServiceScopeFactory serviceScopeFactory, IMapper mapper
         }
     }
 
+    public Task<OrganizationPolicyDetails> GetOrganizationPolicyDetailsByOrgId(Guid orgId, PolicyType policyType) => throw new NotImplementedException();
```

Comment:
The logic for this will be fleshed out in https://github.com/bitwarden/server/pull/6019.